Jenkinsfile

```diff
@@ -87,6 +93,26 @@
     }
 }
 
+def String startLogCatCollector() {
+    sh '''adb logcat -c
+    adb logcat > "logcat.txt" &
+    echo $! > pid
+    '''
+    return readFile("pid").trim()
+}
+
+def stopLogCatCollector(String backgroundPid, boolean archiveLog) {
+    sh "kill ${backgroundPid}"
+    if (archiveLog) {
+        zip([
+            'zipFile': 'logcat.zip',
+            'archive': true,
+            'glob' : '**/logcat.txt'
```

Comment:
such a wildcard can potentially make looking for the file very slow. We know where the file is so let's use the proper path